fuzzers/inprocess/libfuzzer_libpng_accounting/Makefile.toml

```diff
@@ -111,7 +111,10 @@ windows_alias = "unsupported"
 script_runner = "@shell"
 script = '''
 rm -rf libafl_unix_shmem_server || true
-timeout 31s ./${FUZZER_NAME} --cores 0 --input ./corpus | tee fuzz_stdout.log 2>/dev/null || true
+
+# start manager in background, then start one client
+timeout 32s ./${FUZZER_NAME} --cores 0 --input ./corpus 2>/dev/null &
+timeout 31s ./${FUZZER_NAME} --cores 1 --input ./corpus | tee fuzz_stdout.log 2>/dev/null || true
```

Comment:
I added this, otherwise the test doesn't pass. Doesn't either on main btw.

Reply:
Weird, how was this not caught before?

Reply:
Does it still not work now that you fixed the Opt parser btw?

Reply:
I've reverted the change. Most likely something to do with the unwanted code changes. No idea what happened there lol.